http-clients/aws-crt-client/src/main/java/software/amazon/awssdk/http/crt/internal/response/InputStreamAdaptingHttpStreamResponseHandler.java

```diff
@@ -84,7 +88,7 @@ public int onResponseBody(HttpStream stream, byte[] bodyBytesIn) {
 
         writeFuture.whenComplete((result, failure) -> {
             if (failure != null) {
-                failFutureAndCloseConnection(stream, failure);
+                failFutureAndReleaseConnection(stream, failure);
```

Comment:
couple lines down:
```
the bodyBytesIn have not cleared the queues yet, so do let backpressure do its thing.
```

Should it be  "do **not** let backpressure do its thing"?

Reply:
Maybe? I'll update it

Reply:
Oh, maybe I am wrong. It means to let backpress to fill up the window...

It's just confusing, maybe something like:
```
Return zero to not update the flow control window, as the window will be updated once the data completes writing into publisher.
```